pkg/policy/identity.go

```diff
@@ -72,6 +72,16 @@ func (id NumericIdentity) String() string {
 	return id.StringID()
 }
 
+// IsReservedIdentity returns whether id is one of the special reserved identities.
+func (id NumericIdentity) IsReservedIdentity() bool {
+	for reservedIdentity := range ReservedIdentityNames {
```

Comment:
why not `return ReservedIdentityNames[id]`?

Reply:
Heh, good idea. Will fix in next upload.

Reply:
Fixed.